modxcms/revolution
# Update MODX logo and icon

### What does it do?
Updates the MODX logo and icon with those from the 2020 refresh.

NOTE: This removes the old png versions of the icon and logo along with the corresponding references in the stylesheets, as I believe these are no longer needed.

### Why is it needed?
To bring the logo and icon up to date.

### How to test

- Run `grunt build` in `_build/templates/default/`
- Clear browser caches
- View the manager UI to see the new icon
- View the help page in the manager UI to see the new logo

### Related issue(s)/PR(s)
n/a
